python/cuml/neighbors/nearest_neighbors.pyx

```diff
@@ -162,19 +162,21 @@ class NearestNeighbors(Base):
         handles in several streams.
         If it is None, a new one is created.
     algorithm : string (default='brute')
-        The query algorithm to use. Valid options are :
-        - 'brute' for brute-force, slow but produces exact results
-        - 'ivfflat' for inverted file, divide the dataset in partitions
-            and perform search on relevant partitions only
-        - 'ivfpq' for inverted file and product quantization,
-            same as inverted list, in addition the vectors are broken
-            in n_features/M sub-vectors that will be encoded thanks
-            to intermediary k-means clusterings. This encoding provide
-            partial information allowing faster distances calculations
-        - 'ivfsq' for inverted file and scalar quantization,
-            same as inverted list, in addition vectors components
-            are quantized into reduced binary representation allowing
-            faster distances calculations
+        The query algorithm to use. Valid options are:
+
+        - ``'brute'``: for brute-force, slow but produces exact results
```

Comment:
why are the double backticks needed? should all string args be in double backticks?

Reply:
Single and double backticks are shown differently in the documentation. Here is what single backticks look like (italicized, variable spaced):

![image](https://user-images.githubusercontent.com/42954918/106327713-dae58700-623b-11eb-8f4b-09e1e1241ccc.png)

And this is what double backticks look like (bold, monospaced, red):

![image](https://user-images.githubusercontent.com/42954918/106327779-f2247480-623b-11eb-84aa-8b4191d197c3.png)

(Depending on where you are viewing the documentation from, there might be slight differences in color, size, etc.)

Generally in Sphinx, you want to use single backticks when you are referring to some other object that exists (i.e. named argument, class, function, etc.). And you want to use double backticks for inline code (which is different than Markdown which everyone is familiar with). 

For this particular example, I like to use the double backticks around string literals because: 1) it makes them stand out here in the list, 2) string literals aren't referencing any other object and 3) emphasizes that this code can be copy/pasted if needed.

This really only touches the surface since single backticks are technically "interpreted code" which is domain specific and can be modified with labels. For example, you can cross-reference links in the code by using ``:ref:`some-other-link` `` (The `:ref:` part being a label. See [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-documents) for the long list of available labels)